src/rpc/connection/crypto.c

```diff
@@ -446,6 +454,16 @@ int crypto_recv_initiate(struct crypto_context *cc, unsigned char *data)
 
   cc->state = TUNNEL_ESTABLISHED;
 
+  char *pluginkey_base16_encoded = MALLOC_ARRAY((PLUGINKEY_SIZE*2) + 1, char);
```

Comment:
If `plugin_base16_encoded` is allocated on the stack, I need to use `cstring_copy_string`, right? So I don't see any advantage here. Or am I wrong?